package.json

```diff
@@ -25,7 +26,8 @@
   "peerDependencies": {
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-leaflet": "^4.2.0"
+    "react-leaflet": "^4.2.0",
+    "leaflet": "^1.8.0"
```

Comment:
@felipedeboni Thank you a lot for PR.  But I thought react-leaflet already contain  leaflet dependency, do I ?

Reply:
https://github.com/PaulLeCam/react-leaflet/blob/master/package.json#L27

Reply:
@vkruglikov this is the correct one https://github.com/PaulLeCam/react-leaflet/blob/4b71818043fb7308828c17fb927a975833df234c/packages/react-leaflet/package.json#L41, it lists as peerDependency.

However in this project you're consuming the leaflet package in some files apart from types, in this case given it wasn't listed as peer, the bundler was including it in the final bundle.

Before:
<img width="737" alt="Screenshot 2024-07-12 at 11 51 00" src="https://github.com/user-attachments/assets/4c10473e-229a-4019-9b3f-74775fe71d69">

After:
<img width="723" alt="Screenshot 2024-07-12 at 11 51 08" src="https://github.com/user-attachments/assets/ec03ef6c-b851-4a7c-8bf2-d5aaa3cc9ed9">

Reply:
thank you a lot